src/stores/Annotation/Annotation.js

```diff
@@ -647,7 +656,7 @@ export const Annotation = types
       const result = self.serializeAnnotation({ fast: true });
       // if this is new annotation and no regions added yet
 
-      if (!self.pk && !result.length) return;
+      if (!isFF(FF_LSDV_3009) && !self.pk && !result.length) return;
```

Comment:
@nicholasrq I recall you had added this to fix another issue, wouldn't this just be bringing back the original issue by disabling it with feature flag?

Reply:
This code I think was added further back, not sure what the intent was, but I think we just need to make sure of what this guarded against. It may very well not be required, which adding this FF to start the process of removing it would then make sense.